dbt_sugar/core/task/doc.py

```diff
@@ -56,10 +58,123 @@ def run(self) -> int:
+
+        Returns:
+            Dict[str, Any]: with the schema.yml updated.
+        """
+        model_doc_payload: List[Mapping[str, Any]] = [
+            {
+                "type": "confirm",
+                "name": "wants_to_document_model",
+                "message": "Model Description: This is my previously documented model. Document?",
+                "default": True,
+            },
+            {
+                "type": "text",
+                "name": "model_description",
+                "message": "Please write down your description:",
+            },
+        ]
+        user_input = UserInputCollector("model", model_doc_payload).collect()
+        if user_input.get("wants_to_document_model", False):
+            for model in content.get("models", []):
+                if model["name"] == model_name:
+                    model["description"] = user_input["model_description"]
+        return content
+
+    def find_model_in_dbt(self, model_name: str) -> Tuple[Optional[Path], bool]:
+        """
+        Method to find a model name in the DBT depository.
+
+            - If we find the sql of the model but there is no schema we return the Path
+            and False (to create the schema).
+            - If we find the sql of the model and there is schema we return the Path and True.
+
+        Args:
+            model_name (str): model name to find in the DBT repository.
+
+        Returns:
+            Tuple[Optional[Path], bool]: with the path of the sql model
+            and the boolean indicating if the schema.yml exists.
+        """
+        for root, _, files in os.walk(self.repository_path):
+            for file in files:
+                # TODO: Check how to avoid using target to discriminate compiled files in DBT.
+                if file == f"{model_name}.sql" and "target" not in root:
+                    path_file = Path(os.path.join(root, "schema.yml"))
+                    if path_file.is_file():
+                        return path_file, True
+                    else:
+                        return path_file, False
+        return None, False
+
+    def orchestrator_document_model(self, model_name: str, columns_sql: List[str]) -> None:
+        """
+        Orchestrator to fully document a model will:
+
+            - Create or update the columns from the database into the shema.yaml.
+            - Gives the user the posibility to change the model description.
+            - Gives the user the posibility to document the undocumented columns.
+            - Updates all the new columns definitions in all DBT.
+
+        Args:
+            model_name (str): model name that we want to document.
+            columns_sql (List[str]): list of the columns of the model in the database.
+        """
+        content = None
+        path, exists = self.find_model_in_dbt(model_name)
+        if not path:
+            logger.info("Not able to find the model in the repository.")
```

Comment:
May I suggest printing the path as well so that it helps the user ever more with debugging?

Reply:
But the path is None, we can not debug that.

Reply:
The issue is that if a user were to read this they would have no idea what they are looking at?

Maybe print the model name then?

Or actually here's a question: what do you want the user to be able to do when encountering this info message?